### ISSUE TITLE
v1.2.0 displays no pages

### ISSUE THREAD

**Issue by Hu1buerger** [tomoki1207/vscode-pdfviewer#127]
### How to reproduce

Run vscode on macOS, and install pdfviewer with v1.2.0.
Open a folder. Put the pdf from doi.org/10.1145/2503792.2503808 inside. (might or might not be from scihub)

Open the pdf by double clicking on it. 

### What happens

pdfviewer shows that it has 0 pages, and displays a grey background

### What should happen

The ~5 pages should be shown.

#### How I fixed it:

I didn't. Rolling back to v1.1.0 works. The pdf is displayed

**Comment by bob-nomann** [comment]
Same issue (vscode v1.56.2 Universal, macOS Big Sur v11.4), but I found that by interacting with the PDF file the preview in vscode would refresh and be displayed.

Such as opening or editing the file with the built-in macOS Preview app, while the PDF file is also open in vscode.

Further discovered that when live reloading/previewing PDFs, the left sidebar would reopen on each change, which is not ideal.

**Comment by samuela** [comment]
Same issue here. PDF file is attached. Oddly enough, I've found that it does actually show the PDF correctly around 1/10 times.

<img width="1600" alt="Screen Shot 2022-02-09 at 2 29 36 PM" src="https://user-images.githubusercontent.com/226872/153301164-9508e7cc-4a44-4ce8-ada2-a2f41ad284ab.png">

[mnist_mlp_interp_accuracy.pdf](https://github.com/tomoki1207/vscode-pdfviewer/files/8036631/mnist_mlp_interp_accuracy.pdf)

**Comment by jano23** [comment]
I am on Windows and I see the same issue. 
I found a work around and that is to close the grey screen and open the pdf again.
The second time it works, but it gets old having to open each pdf twice!!

**Comment by bulak** [comment]
Same issue here (vscode-pdf v1.2.0) with these combination:
```
Version: 1.66.2
Commit: dfd34e8260c270da74b5c2d86d61aee4b6d56977
Date: 2022-04-11T07:49:24.808Z
Electron: 17.2.0
Chromium: 98.0.4758.109
Node.js: 16.13.0
V8: 9.8.177.11-electron.0
OS: Linux x64 5.13.0-40-generic
```
Perhaps upgrading to the latest `pdf.js` might solve the issue. But manual overwriting gets more cumbersome when multiple remote connections (ssh, kubernetes etc) are involved. Some other PDF viewers have this fixed...

**Comment by bunkerj** [comment]
> I am on Windows and I see the same issue. I found a work around and that is to close the grey screen and open the pdf again. The second time it works, but it gets old having to open each pdf twice!!

Exact same situation for me also on Windows!

**Comment by dfee** [comment]
Of course, same situation over here. Figured I'd follow through and ensure this is an issue.

But, that it's an issue for over 6mo, I'll probably move along to something that's maintained.

**Comment by Ivluengo** [comment]
Same here and it is 9 months now...hope it will be fixed soon as it is pretty anoying... If not I will have to do as @dfee said and move to something else...
